src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
```

Comment:
Use semantic tags instead of `div` ones

Reply:
Will be better to use `ul` and `li` elements as it's more like the list of stars